### PR TITLE
procedures: uninstalling che process update (rhdevdocs-4450)

### PR DESCRIPTION
**Pull request by deerskindoll**
## What does this pull request change?
update of uninstallation steps

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-4450

## Specify the version of the product this pull request applies to
3.1, 3.2, 3.3

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 1]
```diff
@@ -20,26 +20,16 @@ To uninstall an instance of {prod} {prod-ver}:
...
-$ {orch-cli} get checluster --all-namespaces \
-  -o=jsonpath="{.items[*].metadata.namespace}"
+$ {prod-cli} server:delete
 ----
 
-. Remove the {prod-short} instance from the `{prod-namespace}` {orch-namespace}:
-+
-[subs="+attributes,quotes"]
+[TIP]
+
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]:
```suggestion
[TIP]
```

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 2]
```diff
@@ -20,26 +20,16 @@ To uninstall an instance of {prod} {prod-ver}:
...
 ----
-$ {prod-cli} server:delete -n {prod-namespace} # <1>
+Use the `--delete-namespace` flag to remove the {prod} namespace in addition to the instance.
+Use the `--delete:all` flag to remove the {devworkspace} operator in addition to the {prod} namespace and instance.
 ----
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]: We need to use `====` for admonitions. BTW, `TIP` wording doesn't have to be in the imperative mood like the procedure steps, which might help with making it shorter.
```suggestion
====
The `--delete-namespace` option also removes the {prod-short} namespace.

The `--delete-all` option also removes the {devworkspace} Operator, the related resources, and the {prod-short} namespace.
====

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 3]
```diff
@@ -20,26 +20,16 @@ To uninstall an instance of {prod} {prod-ver}:
 
 .Procedure
 
-. Obtain the name of the {prod-short} {orch-namespace}. The default is `{prod-namespace}`.
+* Remove the {prod-short} instance from the `{prod-namespace}` {orch-namespace}:
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]: I suggest getting rid of `{prod-namespace}` here because it might not be necessarily the case (some customers might be using a different namespace), plus it's not important to mention here. (The namespace mattered in the previous version of this procedure because back then it had to be added explicitly, which is no longer the case.)
```suggestion
* Remove the {prod-short} instance:
```

**Comment by max-cx** [comment]
@deerskindoll, I am aware that line 13 was already in the docs and it wasn't part of your changes, but it's worth mentioning here  (1) that we don't normally reference `{prod-ver}` in the docs because it's normally redundant info (the docs are always consistent about one specific version, so it's always `{prod-ver}`) and (2) that `{prod-short}` is more comfortable to read downstream where `{prod}` is quite long (*Red Hat OpenShift Dev Spaces*). Also, senior tech writers have been asking for full-sentence intros.

All in all, I suggest replacing
```
To uninstall an instance of {prod} {prod-ver}:
```
with something like
```
Use `{orch-cli}` to uninstall the {prod-short} instance.
```

**Comment by max-cx** [comment]
Also, I suggest deleting the first prerequisite because it was a prerequisite for a removed step:
```
* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
```

**Comment by deerskindoll** [comment]
@max-cx feedback incorporated

**Review by azatsarynnyy — APPROVED** [review]

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 4]
```diff
@@ -10,36 +10,23 @@
...
-
-include::example$snip_{project-context}-note-uninstalling-in-minikube-cluster.adoc[]
-
-[NOTE] 
+[TIP]
 ====
-If {prod-short} was installed from the OpenShift web console, `{prod-cli}` will not uninstall the {devworkspace} operator.
-To uninstall the {devworkspace} Operator, see link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/odc-about-web-terminal.html#deleting-the-devworkspace-operator-dependency[Deleting the {devworkspace} Operator dependency].
+The `--delete-namespace` option also removes the {prod-short} namespace.
+The `--delete-all` option also removes the {devworkspace} Operator, the related resources, and the {prod-short} namespace.
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]: :) Perhaps try it with a blank line in between (that's why I had it in my suggestion) to break it into two paragraphs?

```suggestion
The `--delete-namespace` option also removes the {prod-short} namespace.

The `--delete-all` option also removes the {devworkspace} Operator, the related resources, and the {prod-short} namespace.
```

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 5]
```diff
@@ -10,36 +10,24 @@
...
-include::example$snip_{project-context}-note-uninstalling-in-minikube-cluster.adoc[]
-
-[NOTE] 
+[TIP]
 ====
-If {prod-short} was installed from the OpenShift web console, `{prod-cli}` will not uninstall the {devworkspace} operator.
-To uninstall the {devworkspace} Operator, see link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/odc-about-web-terminal.html#deleting-the-devworkspace-operator-dependency[Deleting the {devworkspace} Operator dependency].
+The `--delete-namespace` option removes the {prod-short} namespace.
+
+The `--delete-all` option removes the {devworkspace} Operator and the related resources. The {prod-short} namespace isn't removed.
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]: `The {prod-short} namespace isn't removed.`
Because the subject is missing due to the passive voice, this sentence might be initially misread (in terms of readability) as an expectation or instructions.
Prob the best would be to drop this sentence because the first sentence is clear about what gets removed and namespace removal is specified by the other option in the same admonition. If you still feel strongly about making it explicit, you can rephrase it to state that this option does not remove the namespace (but as I said, imo, it's as redundant here as stating for the other option that it does not remove the {devworkspace} Operator and the related resources).

**Review by max-cx — APPROVED** [review]
LGTM with a comment in https://github.com/eclipse-che/che-docs/pull/2491/files#r1023826112.
